apps/HelloPyTorch/Makefile

```diff
@@ -12,30 +12,36 @@ CUDA_OPS =
 else
 HAS_CUDA = 1
 CUDA_OPS = $(BIN)/$(HL_TARGET)/add_cuda_float32.a \
-	       $(BIN)/$(HL_TARGET)/add_cuda_float64.a \
+	   $(BIN)/$(HL_TARGET)/add_cuda_float64.a \
            $(BIN)/$(HL_TARGET)/add_grad_cuda_float32.a \
-	       $(BIN)/$(HL_TARGET)/add_grad_cuda_float64.a
+	   $(BIN)/$(HL_TARGET)/add_grad_cuda_float64.a 
 endif
 
+EXT_LIB=$(BIN)/lib
 PYTHON ?= python3
+PIP ?= pip3
 
 all: test
 
 # Run the PyTorch tests to verify the module is compiled correctly
 # .wrapper is a dummy file whose timestamps allow Make to track the dependency
-# on the Python side of the build
-test: .wrapper
-	@PYTHONPATH=lib:${PYTHONPATH} $(PYTHON) test.py
+# on the Python side of the build 
+
+HALIDE_DISTRIB_PATH=$(shell pwd)/../../distrib
+
+
+test: $(BIN)/.wrapper
+	@PYTHONPATH=$(EXT_LIB):${PYTHONPATH} $(PYTHON) test.py
+
 
 # Build the python PyTorch extension that links against the Halide operators
-.wrapper: $(OPS) $(CUDA_OPS) setup.py
-	@mkdir -p lib
-	@HAS_CUDA=$(HAS_CUDA) \
-		HALIDE_DISTRIB_PATH=$(HALIDE_DISTRIB_PATH) \
-		BIN=$(BIN)/$(HL_TARGET) \
-		PYTHONPATH=lib:${PYTHONPATH} \
-		$(PYTHON) setup.py build -b $(BIN)/python install --install-lib lib
-	@touch .wrapper
+$(BIN)/.wrapper: $(OPS) $(CUDA_OPS) setup.py
+	@mkdir -p $(EXT_LIB)
+	@HAS_CUDA=$(HAS_CUDA) HALIDE_DISTRIB_PATH=$(HALIDE_DISTRIB_PATH) \
```

Comment:
IMHO keeping the one-definition-per-line formatting makes this slightly more readable (or maybe slightly less unreadable)

Reply:
fixed in #96aa9a6